libcontainer/process_linux.go

```diff
@@ -227,6 +241,15 @@ func (p *initProcess) wait() (*os.ProcessState, error) {
 	if p.cmd.SysProcAttr.Cloneflags&syscall.CLONE_NEWPID == 0 {
 		killCgroupProcesses(p.manager)
 	}
+
+	// Call the poststop hooks
+	statePath := p.container.stateFilePath()
+	for _, poststopcmd := range p.poststop {
+		if err := runCmd(poststopcmd, statePath); err != nil {
+			return nil, newSystemError(err)
```

Comment:
The post-stop hooks are “[log any errors but carry on with the teardown][1]”, not “bail after the first error”.

[1]: https://github.com/opencontainers/specs/blob/0c505a55d8e9a6427b97adcee02f40c29e621300/runtime.md#post-stop